.github/workflows/codeflare-sdk-release-sync.yml

```diff
@@ -0,0 +1,80 @@
+---
+# The aim of this GitHub workflow is to update the pipfile to sync with Codeflare-SDK release.
+name: Sync with codeflare-release
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      codeflare_sdk_release_version:
+        required: true
+        description: "Provide the version of the codeflare release you want to update to : "
+      branch:
+        required: true
+        description: "Provide the name of the branch you want to update ex main, vYYYYx etc: "
+
+env:
+  BRANCH_NAME: ${{ github.event.inputs.branch || 'main' }}
+  CODEFLARE_RELEASE_VERSION: ${{ github.event.inputs.codeflare_sdk_release_version }}
+  UPDATER_BRANCH: codeflare-sync-updater-${{ github.run_id }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ env.BRANCH_NAME }}
+
+    # Create a new branch
+    - name: Create a new branch
+      run: |
+        echo ${{ env.UPDATER_BRANCH }}
+        git checkout -b ${{ env.UPDATER_BRANCH }}
+        git push --set-upstream origin ${{ env.UPDATER_BRANCH }}
+
+    - name: Setup Python environment
+      uses: actions/setup-python@v4
+      with:
+        python-version: |
+          3.8
+          3.9
+
+    - name: Install pipenv
+      run: pip install pipenv
+
+    - name: make sync 
+      run: |
+        git config --global user.email "abhijeetdhumal652@gmai.com"
+        git config --global user.name "abhijeet-dhumal" 
```

Comment:
We can use the [codeflare-machine-account](https://github.com/opendatahub-io/codeflare-operator/blob/main/.github/workflows/sync-fork.yaml#L29) here

Reply:
Not sure 100%, though if we use codeflare-machine-account in repo.
you may have to share codeflare-machine-account tokens with us, so that we can set them in this repo.

instead: i would recommend using this:
```
git config --global user.email "github-actions[bot]@users.noreply.github.com"
git config --global user.name "GitHub Actions"
```